Assets/Script/Person.cs

```diff
@@ -32,10 +39,27 @@ void OnCollisionEnter2D(Collision2D coll)
 
             if (infection_chance <= chance_of_infection)
             {
-                Instantiate(infectedPersonPrefab, new Vector3(this.transform.position.x, this.transform.position.y, 0), Quaternion.identity);
+                GameObject infectedPerson = Instantiate(infectedPersonPrefab, new Vector3(this.transform.position.x, this.transform.position.y, 0), Quaternion.identity);
```

Comment:
recoveredPerson ist im moment noch immun, condition entweder um recoveredPerson(clone) erweitern oder this.gameObject.name == "person(Clone)" durch this.gameObject.name  !=infectedPerson(Clone) austauschen